plugins/woocommerce/src/Internal/EmailEditor/WooContentProcessor.php

```diff
@@ -88,7 +107,7 @@ private function capture_woo_content( \WC_Email $wc_email ): string {
 		remove_all_filters( 'woocommerce_email_header' );
 		remove_all_filters( 'woocommerce_email_footer' );
 
-		$woo_content = $wc_email->get_content_html();
+		$woo_content = $wc_email->get_block_editor_email_template_content();
```

Comment:
After the previous conversation I realized that now, with the new `get_block_editor_email_template_content` and copied template with the main content we no longer need to disable and re-enable `woocommerce_email_header` and `woocommerce_email_footer` filters because they are no longer included in the template that is generated via `get_block_editor_email_template_content`. We actually don't event need the method as all we need to call is `$wc_email->get_block_editor_email_template_content();`

I tested it locally and it seems to work without them. This is a nice side-effect of your idea with the copy of general email content. 👍 

I'm sorry that I didn't realize it in the first or second review. Could you please remove the manipulation with filters?

Reply:
No problem. I missed this during the last cleanup as well. Done in https://github.com/woocommerce/woocommerce/pull/56567/commits/472982c792130e4a62e92253f593a64595afa7d6